config/software/inspec.rb

```diff
@@ -32,8 +32,10 @@
 build do
   env = with_standard_compiler_flags(with_embedded_path)
 
-  bundle "install --without test integration tools maintenance", env: env
+  bundle "config set --local without test integration tools maintenance", env: env
+  bundle "install", env: env
 
+  gem "build inspec.gemspec", env: env
```

Comment:
I don't think we need/want to duplicate this line?

Reply:
I have added this to resolve inspec gem installation. Earlier it was failing because it could not find a valid gem 'inspec-core' (= 4.41.20)
 https://buildkite.com/chef-oss/chef-omnibus-software-main-verify/builds/1156